### PR TITLE
Fixed situations when reload window failed tests in ee

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ContextServicesTestCase.java [hunk 1]
```diff
@@ -91,18 +92,19 @@ public void addContextServiceInGUI() {
         Editor editor = wizard.getEditor();
         editor.text("name", name);
         editor.text(JNDI_NAME, JNDI_VALID);
-        boolean result = wizard.finish();
+        wizard.clickSave();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ContextServicesTestCase.java]: In meantime https://github.com/zebrik/testsuite/commit/4e46b07e7ca2d8543f26915d9b9bb92d3f7b87f6 was merged so you can use `WizardWindow.finishAndDismissReloadRequiredWindow()` instead.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ExecutorTestCase.java [hunk 2]
```diff
@@ -167,18 +168,19 @@ public void addExecutorInGUI() {
         Editor editor = wizard.getEditor();
         editor.text("name", name);
         editor.text(JNDI_NAME, JNDI_VALID);
-        boolean result = wizard.finish();
+        wizard.clickSave();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ExecutorTestCase.java]: In meantime https://github.com/zebrik/testsuite/commit/4e46b07e7ca2d8543f26915d9b9bb92d3f7b87f6 was merged so you can use `WizardWindow.finishAndDismissReloadRequiredWindow()` instead.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ScheduledExecutorTestCase.java [hunk 3]
```diff
@@ -151,18 +152,19 @@ public void addExecutorInGUI() {
         Editor editor = wizard.getEditor();
         editor.text("name", name);
         editor.text(JNDI_NAME, JNDI_VALID);
-        boolean result = wizard.finish();
+        wizard.clickSave();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ScheduledExecutorTestCase.java]: In meantime https://github.com/zebrik/testsuite/commit/4e46b07e7ca2d8543f26915d9b9bb92d3f7b87f6 was merged so you can use `WizardWindow.finishAndDismissReloadRequiredWindow()` instead.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ThreadFactoryTestCase.java [hunk 4]
```diff
@@ -100,18 +101,19 @@ public void addThreadFactoryInGUI() {
         Editor editor = wizard.getEditor();
         editor.text("name", name);
         editor.text(JNDI_NAME, JNDI_VALID);
-        boolean result = wizard.finish();
+        wizard.clickSave();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ThreadFactoryTestCase.java]: In meantime https://github.com/zebrik/testsuite/commit/4e46b07e7ca2d8543f26915d9b9bb92d3f7b87f6 was merged so you can use `WizardWindow.finishAndDismissReloadRequiredWindow()` instead.

**Comment by honza-kasik** [comment]
Added methods from #138

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ContextServicesTestCase.java [hunk 5]
```diff
@@ -8,6 +8,7 @@
 import org.jboss.hal.testsuite.fragment.ConfigFragment;
 import org.jboss.hal.testsuite.fragment.formeditor.Editor;
 import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+import org.jboss.hal.testsuite.util.Console;
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ContextServicesTestCase.java]: Remove unused import.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ExecutorTestCase.java [hunk 6]
```diff
@@ -8,6 +8,7 @@
 import org.jboss.hal.testsuite.fragment.ConfigFragment;
 import org.jboss.hal.testsuite.fragment.formeditor.Editor;
 import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+import org.jboss.hal.testsuite.util.Console;
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ExecutorTestCase.java]: Remove unused import.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ScheduledExecutorTestCase.java [hunk 7]
```diff
@@ -8,6 +8,7 @@
 import org.jboss.hal.testsuite.fragment.ConfigFragment;
 import org.jboss.hal.testsuite.fragment.formeditor.Editor;
 import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+import org.jboss.hal.testsuite.util.Console;
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ScheduledExecutorTestCase.java]: Remove unused import.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ThreadFactoryTestCase.java [hunk 8]
```diff
@@ -8,6 +8,7 @@
 import org.jboss.hal.testsuite.fragment.ConfigFragment;
 import org.jboss.hal.testsuite.fragment.formeditor.Editor;
 import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+import org.jboss.hal.testsuite.util.Console;
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/ee/ThreadFactoryTestCase.java]: Remove unused import.

**Comment by honza-kasik** [comment]
Removed unused imports.

**Comment by zebrik** [comment]
LGTM. Thx.

**Pull request merged**